qa/load-testing/main.py

```diff
@@ -0,0 +1,24 @@
+from argparse import ArgumentParser
+import os
+import requests
+
+DOCKER_HOST_IP=os.getenv('DOCKER_HOST_IP')
+assert DOCKER_HOST_IP != None, "Failed to get DOCKER_HOST_IP from environment variable"
+UPLOAD_URL = "http://%s:8080/upload/bsmlog" % DOCKER_HOST_IP
+UPLOAD_FILE = "../../data/bsmLogDuringEvent.gz"
```

Comment:
Would be nice to provide a directory and have the code to upload all the files in it.